values.yaml

```diff
@@ -2,10 +2,12 @@ namespace: default
 env: dev
 k8s_1dot8_or_above: true
 has_static_volumes: true
+arch: x86_64
 services:
   expose_node_port: true
 docker:
-  registry: docker.io
+  registry: ffdl.ibm.com
```

Comment:
This changes our official Docker image location.

Reply:
Sorry my mistake. This was not supposed to be there. Will fix it.